docs/src/App.js

```diff
@@ -4,167 +4,19 @@
-  }
-
-  div {
-    font-size: 14px;
-    opacity: 0.5;
-    margin-bottom: 0.5rem;
-  }
-`;
-
-const MainContainer = styled.main`
-  margin-left: 240px;
-  padding: 0 2rem 2rem;
-  @media only screen and (max-width: 680px) {
-    margin-left: 0;
-    margin-top: 4rem;
-  }
-`;
-
-const Main = styled.div`
-  max-width: 1040px;
-  margin: 0 auto;
-`;
-
-const MobileNavigationToggle = styled.a`
-  display: none;
-  padding: 2rem;
-  cursor: pointer;
-  &:hover {
-    opacity: 0.5;
-  }
-  @media only screen and (max-width: 680px) {
-    display: block;
-  }
-`;
-
-function DefaultMain({ name, ...rest }) {
-  return (
-    <div>
-      <h1>WIP</h1> Coming soon...
-    </div>
-  );
-}
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import Landing from "./Landing";
+import Docs from "./Docs";
 
 export default function App() {
-  const [isMobileNavOpen, toggleMobileNav] = useState(0);
   return (
     <MDXProvider components={{}}>
       <Router>
         <React.Fragment>
-          <SideBar>
-            <LogoLink to="/">
-              <Logo width={160} />
-            </LogoLink>
-            <MobileNavigationToggle onClick={() => toggleMobileNav(!isMobileNavOpen)}>
-              {isMobileNavOpen ? "↑" : "↓"}
-            </MobileNavigationToggle>
-            <Navigation isMobileNavOpen={isMobileNavOpen}>
-              {routes.map((route) => (
-                <Section key={route.path}>
-                  <div>{route.name}</div>
-                  <ul>
-                    {route.subRoutes.map((subRoute) => (
-                      <li key={subRoute.path}>
-                        <Link to={`${route.path}${subRoute.path}`} onClick={() => toggleMobileNav(false)}>
-                          {subRoute.name || subRoute.main}
-                        </Link>
-                      </li>
-                    ))}
-                  </ul>
-                </Section>
-              ))}
-            </Navigation>
-          </SideBar>
-          <MainContainer>
-            <Main>
-              <Route exact path="/" render={() => <Redirect to="/guides" />} />
-              {routes.map((route) => (
-                <React.Fragment key={route.path}>
-                  <Route
-                    exact
-                    path={route.path}
-                    render={() => <Redirect to={`${route.path}${route.subRoutes[0].path}`} />}
-                  />
-                  {route.subRoutes.map((subRoute) => (
-                    <div className="markdown-body" key={subRoute.path}>
-                      <Route
-                        exact={subRoute.exact}
-                        path={`${route.path}${subRoute.path}`}
-                        component={componentList[subRoute.main] || DefaultMain}
-                      />
-                    </div>
-                  ))}
-                </React.Fragment>
-              ))}
-            </Main>
-          </MainContainer>
+          <Route exact path="/" component={Landing} />
```

Comment:
Switch might be a good option here, so we can catch all routes that don't have `/docs` and render the landing page. 
```
     <Switch>
            <Route path="/docs" component={Docs} />
            <Route path="/" component={Landing} />
          </Switch>
```